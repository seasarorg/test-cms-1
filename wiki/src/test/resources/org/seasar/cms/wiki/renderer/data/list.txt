- リスト1
- リスト2

+ 数字付きリスト1
子要素がパラグラフ系
+ 数字付きリスト2
++ 数字付き二段目1
++ 数字付き二段目2
+++ 数字付き三段目3
++ 数字付き戻る
+ 数字付き戻る
++ 復帰

-- 二段リスト1
-- 二段リスト2
-- 二段リスト3

- 多段リスト1
-- 多段リスト子要素レベル1
--- 多段リスト子要素レベル2
- 多段リスト2

-レベル1
--レベル2
---レベル3
-レベル1
--レベル2

- 空要素込み
-
-
- 空要素終わり

-テスト～①
--テスト～①
+テスト～①
+-テスト～①
-+テスト～①