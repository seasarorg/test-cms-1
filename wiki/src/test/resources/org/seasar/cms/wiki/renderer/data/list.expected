<ul><li> リスト1</li><li> リスト2</li></ul>
<ol>
<li> 数字付きリスト1<p>子要素がパラグラフ系</p>
</li><li> 数字付きリスト2</li><ol>
<li> 数字付き二段目1</li><li> 数字付き二段目2</li><ol>
<li> 数字付き三段目3</li></ol>
<li> 数字付き戻る</li></ol>
<li> 数字付き戻る</li><ol>
<li> 復帰</li></ol>
</ol>
<ul>
<li> 二段リスト1</li><li> 二段リスト2</li><li> 二段リスト3</li></ul>
<ul>
<li> 多段リスト1</li><ul>
<li> 多段リスト子要素レベル1</li><ul>
<li> 多段リスト子要素レベル2</li></ul>
</ul>
<li> 多段リスト2</li></ul>
<ul>
<li>レベル1</li><ul>
<li>レベル2</li><ul>
<li>レベル3</li></ul>
</ul>
<li>レベル1</li><ul>
<li>レベル2</li></ul>
</ul>
<ul>
<li> 空要素込み</li><li> 空要素終わり</li></ul>
<ul>
<li>テスト～①</li><ul>
<li>テスト～①</li></ul>
</ul>
<ol>
<li>テスト～①</li><li>-テスト～①</li></ol>
<ul>
<li>+テスト～①</li></ul>
